yarn-project/bb-prover/src/prover/bb_prover.ts

```diff
@@ -148,6 +153,7 @@ export class BBNativeRollupProver implements ServerCircuitProver {
    * @param inputs - Inputs to the circuit.
    * @returns The public inputs of the parity circuit.
    */
+  @trackSpan('BBNativeRollupProver.getBaseParityProof', { [Attributes.PROTOCOL_CIRCUIT_NAME]: 'root-parity' })
```

Comment:
```suggestion
  @trackSpan('BBNativeRollupProver.getRootParityProof', { [Attributes.PROTOCOL_CIRCUIT_NAME]: 'root-parity' })
```

I wonder if we could set up a custom eslint plugin to catch these